cmd/gokr-packer/packer.go

```diff
@@ -889,12 +919,16 @@ func logic() error {
 		}
 	}
 
-	newInstallation := updateflag.NewInstallation()
 	p := pack{
 		Pack: packer.NewPackForHost(*hostname),
 	}
-	p.Pack.UsePartuuid = newInstallation
-	p.Pack.UseGPTPartuuid = newInstallation
+
+	useGPT := updateflag.NewInstallation() && !*mbrOnlyWithoutGpt
+
+	p.Pack.UsePartuuid = useGPT
```

Comment:
UsePartuuid isn’t tied to GPT support, I think? This should remain `p.Pack.UsePartuuid = newInstallation`